test/integration/puppetlabs/services/jruby/jruby_locking_test.clj

```diff
@@ -23,16 +32,15 @@
     [jruby-service/jruby-puppet-pooled-service
      profiler/puppet-profiler-service]
     (jruby-service-test-config 1)
-    (let [jruby-service (tk-app/get-service app :JRubyPuppetService)
-          lock (jruby-testutils/get-lock-from-app app)]
-
+    (jruby-testutils/wait-for-jrubies app)
```

Comment:
This is unnecessary, since there is only a single JRuby in-use by this test, right?